app/views/layouts/_navigation.html.erb

```diff
@@ -33,14 +33,17 @@
   </button>
   <div class="collapse navbar-collapse" id="navbarText">
     <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/obituaries/community">Community Obituaries</a>
+      </li>
       <li class="nav-item">
         <a class="nav-link" href="/obituaries/covid-19">COVID-19 Obituaries</a>
       </li>
       <li class="nav-item">
         <a class="nav-link" href="/obituaries/recent">Recently Published</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" href="/obituaries/community">Community Obituaries</a>
+        <a class="nav-link" href="/resources">Additional Resources</a>
```

Comment:
Added resources page to the nav bar. It is a little crowded and we can discuss options for dealing with that if you want.